### PR TITLE
Adapt theme for new presenter system

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Makes use of the new presenters feeding data to FO, allows use of multiple image formats. This PR is required for this to work - https://github.com/PrestaShop/PrestaShop/pull/32634
| Type?             | new feature
| BC breaks?        | yes
| Deprecations?     | no
| Fixed ticket?     | 
| Sponsor company   | 
| How to test?      | Install https://github.com/PrestaShop/PrestaShop/pull/32634, install this, check that it behaves the same.

### CODE REVIEW

**Review by tleon — APPROVED** [review]
LGTM

File: templates/catalog/manufacturers.tpl [hunk 1]
```diff
@@ -22,4 +22,21 @@
...
+     {block name='brand_miniature'}
+       <ul>
+         {foreach from=$brands item=brand}
+           {include file='catalog/_partials/miniatures/brand.tpl' brand=$brand}
+         {/foreach}
+       </ul>
+     {/block}
+ 
+   </section>
+ {/block}
```
Comment (M0rgan01) [inline templates/catalog/manufacturers.tpl]:
```suggestion
{extends file=$layout}
{block name='content'}
 <section id="main">

   {block name='brand_header'}
     <h1>{l s='Brands' d='Shop.Theme.Catalog'}</h1>
   {/block}

   {block name='brand_miniature'}
     <ul>
       {foreach from=$brands item=brand}
         {include file='catalog/_partials/miniatures/brand.tpl' brand=$brand}
       {/foreach}
     </ul>
   {/block}

 </section>
{/block}
```

The indentation is not good it seems to me

**Comment by Hlavtox** [comment]
@M0rgan01 @tleon Fixed. :-)

**Review by M0rgan01 — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Review by florine2623 — APPROVED** [review]
@Hlavtox ,

Tested with https://github.com/PrestaShop/PrestaShop/pull/32634

https://github.com/PrestaShop/PrestaShop/assets/16019289/61b8bdc7-204a-4554-a918-9dd962e5a4ad

LGTM. It is QA ✅ !
Thanks!

**Comment by nicosomb** [comment]
Thank you @Hlavtox !

**Pull request merged**